.github/workflows/container-build.yml

```diff
@@ -0,0 +1,91 @@
+name: Build/Release Container Image
+
+on:
+  push:
+    branches: [ "main", "ci-*" ]
```

Comment:
What's the `ci-*` branch? Does that get created by the docker action?

Reply:
It is just to test CI thing, i.e. any branch with the `ci-` prefix can be used to develop/test new CI things.